### PR TITLE
Override moment formats

### LINKED ISSUE #2099: Override moment formats
**Is your feature request related to a problem? Please describe.**
Incorrect moment formats. i.e in Russian expected `8 июля 2019` what moment currently displays `8 июля 2019 г.` We need to remove the `r.`.

**Describe the solution you'd like**
Update moment to have the correct format for the following services.

| Service | Current | Expected |
|--------|---------|--------|
| Brasil | 8 de Julho de 2019 | 8 Julho 2019 |
| Russian | 8 июля 2019 г. | 8 июля 2019 |
| Serbian | 8. јул 2019 | 8 јул 2019 |
| Ukrainian | 8 липня 2019 р. | 8 липня 2019 |

More details WSFRONTPGS-197

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #2099

**Overall change:** 
Update moment to have the correct format for `Brasil`, `Russian`, `Serbian` and `Ukrainian`.

**Code changes:**
- Add `pt-br`, `ru`, `sr`, `sr-cyrl` and `uk` files to `psammead-locales`.
- Use `moment.updateLocale` to override the moment locales.
- Bump `psammead-locales`
- Update README

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.3.1 | [PR#2114](https://github.com/bbc/psammead/pull/2114) Add some overrides for Brasil, Russian, Serbian and Ukrainian |
```
Comment (ghost) [inline packages/utilities/psammead-locales/CHANGELOG.md]:
```suggestion
| 2.4.0 | [PR#2114](https://github.com/bbc/psammead/pull/2114) Add some overrides for Brasil, Russian, Serbian and Ukrainian |
```

File: packages/utilities/psammead-locales/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-locales",
-  "version": "2.3.0",
+  "version": "2.3.1",
```
Comment (ghost) [inline packages/utilities/psammead-locales/package.json]:
```suggestion
  "version": "2.4.0",
```

File: packages/utilities/psammead-locales/package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-locales",
-  "version": "2.3.0",
+  "version": "2.3.1",
```
Comment (ghost) [inline packages/utilities/psammead-locales/package-lock.json]:
```suggestion
  "version": "2.4.0",
```

**Review by dr3 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**